activation/activation_test.go

```diff
@@ -186,73 +72,75 @@ func newActivationTx(
 	nipost *types.NIPost,
 ) *types.VerifiedActivationTx {
 	challenge := newChallenge(sequence, prevATX, positioningATX, pubLayerID, cATX)
-	atx := newAtx(t, challenge, sig, nipost, numUnits, coinbase)
+	atx := newAtx(t, sig, nodeID, challenge, nipost, numUnits, coinbase)
 	vAtx, err := atx.Verify(startTick, numTicks)
 	require.NoError(t, err)
 	return vAtx
 }
 
-type LayerClockMock struct {
-	mu           sync.Mutex
-	currentLayer types.LayerID
-}
-
-func (l *LayerClockMock) LayerToTime(types.LayerID) time.Time {
-	return time.Time{}
-}
-
-func (l *LayerClockMock) GetCurrentLayer() types.LayerID {
-	l.mu.Lock()
-	defer l.mu.Unlock()
-	return l.currentLayer
-}
-
-func (l *LayerClockMock) AwaitLayer(types.LayerID) chan struct{} {
-	ch := make(chan struct{})
-	go func() {
-		time.Sleep(500 * time.Millisecond)
-		close(ch)
-	}()
-	return ch
-}
+type testAtxBuilder struct {
+	*Builder
+	cdb         *datastore.CachedDB
+	sig         *signing.EdSigner
+	nodeID      *types.NodeID
```

Comment:
NIT, I think there is no need for a pointer here
```suggestion
	nodeID      types.NodeID
```

Reply:
done